.github/workflows/version-update.yml

```diff
@@ -0,0 +1,40 @@
+name: Update package version for release & hotfix branches
+
+on:
+  create:
+    branches: [release/*, hotfix/*]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - name: update package version
+        id: vars
+        run: |
+          BRANCH_NAME="${GITHUB_REF#refs/heads/}"
+          PACKAGE_VERSION="${GITHUB_REF##*/}"
+          echo ::set-output name=branch::${BRANCH_NAME}
+          echo ::set-output name=version::${PACKAGE_VERSION}
+          if [[ $PACKAGE_VERSION =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]
+          then
+            git config user.name 'github-actions[bot]'
+            git config user.email 'github-actions[bot]@users.noreply.github.com'
+            npm version ${PACKAGE_VERSION}
+            git push -u origin HEAD:"dev/update-version-${PACKAGE_VERSION}"
```

Comment:
I suppose this push could fail if the branch name already exists, however I'm not sure if it's worth handling that edge case. If that branch already exists, we can assume that it updates the version number

Reply:
yeah, if the branch exist, then the pr is probably still open (since we usually close the pr and delete the branch together). I think it should be sufficient. Perhaps if this edge case comes up often, then we can address it then!